packages/components/src/Page/Page.tsx

```diff
@@ -194,7 +268,846 @@ export function Page({
+                />
+
+                <Text variation="subdued">
+                  19) Simple opts — section actions and flat actions (no
+                  options)
+                </Text>
+                <Autocomplete
+                  version={2}
+                  value={s5Value}
+                  onChange={setS5Value}
+                  inputValue={s5Input}
+                  onInputChange={setS5Input}
+                  menu={[
+                    {
+                      type: "options",
+                      options: [],
+                      actionsBottom: [
+                        {
+                          type: "action",
+                          id: "add7",
+                          label: "Empty Options Action",
+                          onClick: () => {},
+                        },
+                      ],
+                    },
+                    {
+                      type: "section",
+                      id: 19,
+                      label: "Numbers",
+                      options: simpleOptions,
+                      actionsBottom: [
+                        {
+                          type: "action",
+                          id: "add8",
+                          label: "Add",
+                          onClick: () => {},
+                        },
+                      ],
+                    },
+                  ]}
+                  filterOptions={(o, i) =>
+                    o.label.toLowerCase().includes(i.toLowerCase())
+                  }
+                  getOptionLabel={o => o.label}
+                  getOptionValue={o => o.id}
+                />
+
+                <Text variation="subdued">20) Empty options</Text>
+                <Autocomplete
+                  version={2}
+                  value={s5Value}
+                  onChange={setS5Value}
+                  inputValue={s5Input}
+                  onInputChange={setS5Input}
+                  menu={[{ type: "options", options: [] }]}
+                  filterOptions={(o, i) =>
+                    o.label.toLowerCase().includes(i.toLowerCase())
+                  }
+                  getOptionLabel={o => o.label}
+                  getOptionValue={o => o.id}
+                />
+
+                <Text variation="subdued">20) Custom Input</Text>
+                <Autocomplete
+                  version={2}
+                  value={c1Value}
+                  onChange={setC1Value}
+                  inputValue={c1Input}
+                  onInputChange={setC1Input}
+                  menu={[{ type: "options", options: simpleOptions }]}
+                  filterOptions={(o, i) =>
+                    o.label.toLowerCase().includes(i.toLowerCase())
+                  }
+                  renderInput={({ inputRef, inputProps }) => (
+                    <InputText
+                      {...inputProps}
+                      ref={inputRef}
+                      suffix={{ icon: "arrowDown" }}
```

Comment:
as an aside, I would love a more open API on InputText

it currently has `children` as a prop on v2, however it blow up if you provide it.